aten/src/ATen/native/quantized/cpu/qnnpack/CMakeLists.txt

```diff
@@ -391,6 +391,12 @@ elseif(NOT TARGET fxdiv AND USE_SYSTEM_FXDIV)
 endif()
 target_link_libraries(pytorch_qnnpack PRIVATE fxdiv)
 
+# -- [ CMake-4 compat mode
```

Comment:
@malfet we need to get rid of qnnpack, but i wont be surprised of long tail of models somewhere wanting to use